rosetta-2021.html

```diff
@@ -0,0 +1,236 @@
+<script>
+  // Rule of thumb: put a semicolon at the end of every line that doesn't end in a curly
+
+  // Simple output
+  // =============
+
+  // puts "howdy!"
+
+  // alert("howdy!");
+
+
+  // Declaring local variables
+  // =========================
+
+  // name = "Raghu"
+  // puts "howdy, #{name}!"
+
+  // var name = "Raghu";
```

Comment:
In 2021, is this supposed to be `let` instead of `var`?

Reply:
Hmm, I think generally yes. `let` is a block-scoped local variable, whereas I think `var` is global. But again `let` is ES6 which isn't supported on older browsers (unless you're using a transpiler).

Reply:
I didn't link to the docs for the specifics but [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) and [`const`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) are scoped while [`var`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var) is global.